public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+    }
+  },
+  "vagabond": {
+    "adventurer": {
+      "name": "సాహసికుడు",
+      "action": "మెరుగుపరచండి",
+      "effect": "క్వెస్ట్ చర్య తీసుకునేటప్పుడు ఒక్కో మలుపుకు ఒకసారి, మీరు ఒక అయిపోని వస్తువును ఏదైనా ఇతర వస్తువుగా పరిగణించవచ్చు. అన్వేషణను పూర్తి చేయడానికి మీరు దాన్ని ఎగ్జాస్ట్ చేసినప్పుడు, ఆ అంశాన్ని కూడా పాడు చేయండి."
+    },
+    "arbiter": {
+      "name": "మధ్యవర్తి",
+      "action": "రక్షకుడు",
+      "effect": "యుద్ధంలో పాచికలు వేయడానికి ముందు, డిఫెండర్ యుద్ధం యొక్క క్లియరింగ్‌లో ఆర్బిటర్‌ను చేర్చుకోవచ్చు. ఆర్బిటర్ ఒక విక్టరీ పాయింట్‌ని స్కోర్ చేస్తాడు మరియు డిఫెండర్ యొక్క గరిష్ట రోల్డ్ హిట్‌లకు అతని దెబ్బతినని <Sword/> మొత్తాన్ని జోడిస్తుంది. మధ్యవర్తి తనను తాను చేర్చుకోలేరు లేదా తనకు వ్యతిరేకంగా నమోదు చేసుకోలేరు."
+    },
+    "harrier": {
+      "name": "హారియర్",
+      "action": "నెమ్మదిగా కదలు",
+      "effect": "మీ వాగాబాండ్ బంటుని <i>(ఇతర ముక్కలు కాదు)</i>ని ఏ <Boot/> అయిపోకుండా మ్యాప్‌లోని ఏదైనా క్లియరింగ్ <i>(విరోధి కూడా)</i>కి తరలించడానికి <టార్చ్/>ని ఎగ్జాస్ట్ చేయండి."
+    },
+    "ranger": {
+      "name": "రేంజర్",
+      "action": "సురక్షితమైన దాక్కునేచోటు",
+      "effect": "మూడు వస్తువులను రిపేర్ చేయడానికి ఒక <Torch/> ఖాళీ చేయండి. తర్వాత, వెంటనే డేలైట్‌ని ముగించి సాయంత్రం ప్రారంభించండి."
+    },
+    "ronin": {
+      "name": "రోనిన్",
+      "action": "వేగంగా దెబ్బవేయు",
+      "effect": "యుద్ధంలో <i>(రోలింగ్ తర్వాత)</i> అదనపు హిట్‌ని ఎదుర్కోవడానికి మీరు <Sword/>ని ఖాళీ చేయవచ్చు."
+    },
+    "scoundrel": {
+      "name": "దుష్టుడు",
+      "action": "మండుతున్న భూమి",
+      "effect": "<Torch/>ని ఖాళీ చేసి, దానిని మీ క్లియరింగ్‌లో ఉంచండి. ఆ క్లియరింగ్ నుండి అన్ని శత్రువు ముక్కలను తొలగించండి. <Torch/>తో ముక్కలను ఉంచడం లేదా క్లియరింగ్‌లోకి తరలించడం సాధ్యం కాదు. <i>(మీరు ఈ క్లియరింగ్‌లో ఉంటారు. మీరు బయటకు వెళ్లిన తర్వాత, మీరు తిరిగి లోపలికి వెళ్లలేరు. \"ఫేవర్ ఆఫ్ ది...\" కార్డ్‌తో <Torch/> తీసివేయబడదు ఎందుకంటే ఇది శత్రువు ముక్క కాదు. )</i>"
+    },
+    "thief": {
+      "name": "దొంగ",
+      "action": "దొంగిలించు",
+      "effect": "మీ క్లియరింగ్‌లో ఉన్న ఏ ఆటగాడు నుండి అయినా యాదృచ్ఛిక కార్డ్‌ని తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి."
+    },
+    "tinker": {
+      "name": "లోహపు పాత్రలను బాగు చేయువాడు",
+      "action": "రోజు కూలీ",
+      "effect": "మీ క్లియరింగ్‌కు సరిపోయే సూట్‌ను డిస్కార్డ్ పైల్ నుండి తీసుకోవడానికి ఒక <Torch/>ని ఖాళీ చేయండి. <i>(మీరు ఎప్పుడైనా బర్డ్ కార్డ్ తీసుకోవచ్చు.)</i>"
+    },
+    "vagrant": {
+      "name": "ఒకచోట నుండి మరియొక చోటికి మారు",
```

Comment:
This vagabond name is simply too long to fit in the interface. I would advise against trying to perfectly capture the meaning of the word "vagrant" and instead opt for a nice single word that could describe someone who is a trouble maker. Conciseness is far more important than accuracy here, you just want a fun word that people might associate with how this vagabond plays.